pkg/reconciler/isbsvc/controller.go

```diff
@@ -41,7 +41,9 @@ import (
 )
 
 const (
-	finalizerName = dfv1.ControllerISBSvc
+	finalizerName = "numaflow.numaproj.io/" + dfv1.ControllerISBSvc
+	// TODO: clean up the deprecated finalizer in v1.7
```

Comment:
```suggestion
	// DEPRECATE(1.7): clean up the deprecated finalizer in v1.7
```
Is this a simpler search pattern?

Reply:
Might get confused with `Deprecated` annotation. We could search `Deprecated` in the future.